scintillometry/channelize.py

```diff
@@ -2,8 +2,8 @@
 
 import operator
 
-from ..base.base import TaskBase
-from ..fourier import get_fft_maker
+from .base import TaskBase
+from .fourier import get_fft_maker
```

Comment:
Shouldn't this import be from `.helper.fourier`?

Reply:
For usability, I like being able to go `from scintillometry.fourier import get_fft_maker` rather than `from scintillometry.helpers.fourier import get_fft_maker`, or do you think it's too early to make these considerations?